extensions/wikia/MyHome/data/ActivityFeedAPIProxy.php

```diff
@@ -25,7 +25,7 @@ public function get($limit, $start = null) {
 		$this->APIparams['rclimit'] = $limit;
 		$api = new ApiMain(new FauxRequest($this->APIparams));
 		$api->execute();
```

Comment:
Changes in this file related to [SOC-452](https://wikia-inc.atlassian.net/browse/SOC-452)